lib/rbs/definition_builder.rb

```diff
@@ -239,6 +239,19 @@ def build_singleton0(type_name)
             definition.class_variables.merge!(defn.class_variables)
           end
 
+          one_ancestors.each_included_module do |mod|
+            included_module = env.class_decls[mod.name] or raise "Unknown name for one_ancestors.included_module: #{type_name}"
+            included_module.decls.each do |decl|
+              decl.decl.annotations.each do |annotation|
+                if annotation.string.start_with? "autoextend:"
```

Comment:
Should we support type arguments for modules? If so, this code should be updated more.

Reply:
Forget the generics for now.
It shouldn't happen frequently.